modules/tasks/tasks/src/main/resources/webapp/partials/widgets/content-editable-period.html

```diff
@@ -0,0 +1,42 @@
+<input type="hidden" ng-model="data.value" period-amount>
+<button class="btn-sm btn btn-default">
+    {{msg('task.btn.changePeriod')}}
+</button>
+<div contenteditable ng-model="data.value" droppable class="actionField period-input period-field word-wrap input-extra-fluid" data-index="{{index}}" data-type="{{data.type}}">
+    <span ng-bind-html-unsafe="data.value" ng-model="data.value" class="html">
+        <h1 contentEditable="false">
+            <span class="editable"></span>
+        </h1>
+    </span>
+</div>
+<div id="periodModal" class="tasks modal fade" tabindex="-1" role="dialog" aria-hidden="true" data-index="{{index}}" data-action="{{action}}">
```

Comment:
This div and the under [this link](https://github.com/motech/motech/blob/master/platform/mds/mds-web/src/main/resources/webapp/partials/widgets/field-basic-defaultValue-period.html#L7) are basically the same. What about placing this code in a separate .html file and loading it via angular directive? Here is a working example - [link](https://github.com/motech/motech/blob/master/platform/mds/mds-web/src/main/resources/webapp/js/directives.js#L3363).